src/Cards.h

```diff
@@ -0,0 +1,130 @@
+
+#pragma once
+#ifndef EIGHT_MINUTE_EMPIRE_CARDS_H
+#define EIGHT_MINUTE_EMPIRE_CARDS_H
```

Comment:
I might be mistaken, but `pragma once` and `ifndef EIGHT_MINUTE_EMPIRE_CARDS_H` are equivalent, so you don't actually need both.